### DISCUSSION TITLE
Trino-Hive connector simple authentification

### DISCUSSION THREAD

**Discussion by Vadrrrr** [trinodb/trino#24486] [Q&A] [answered]
Hi, does the Trino-Hive connector support simple authentication (username and password)? I only see Kerberos mentioned in the docs. Are there plans to add simple authentication? If not, we’re ready to contribute and add this functionality ourselves.

**Comment by hashhar** [comment, marked as answer]
Trino itself supports basic auth (username + password).

Do you mean authentication when talking to HMS or HDFS in this case? From what I know neither HDFS nor HMS support username + password as an authentication mechanism.
